### ISSUE TITLE
Add VUID-VkCopyAccelerationStructureToMemoryInfoKHR-dst-03561

### ISSUE THREAD

**Issue by arno-lunarg** [KhronosGroup/Vulkan-ValidationLayers#7481]
TL;DR:
Adding validation for this VU is of low priority

**VUID-VkCopyAccelerationStructureToMemoryInfoKHR-dst-03561**
The memory pointed to by dst must be at least as large as the serialization size of src, as reported by [vkWriteAccelerationStructuresPropertiesKHR](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkWriteAccelerationStructuresPropertiesKHR.html) or [vkCmdWriteAccelerationStructuresPropertiesKHR](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCmdWriteAccelerationStructuresPropertiesKHR.html) with a query type of VK_QUERY_TYPE_ACCELERATION_STRUCTURE_SERIALIZATION_SIZE_KHR

Validation for this VU will only be done when the struct is used in `vkCmdCopyAccelerationStructureToMemoryKHR`, as for the host version, making sure that the CPU memory has been allocated properly is out of the scope of the VVL.

Doing a call to vkCmdWriteAccelerationStructuresPropertiesKHR demands to setup a query pool, dispatching the query, submit the command buffer to a queue, wait for its completion, and finally read back the result.
Falling back on the host version requires the [accelerationStructureHostCommands](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#features-accelerationStructureHostCommands) feature, a feature that is likely to not be enabled when one uses `vkCmdCopyAccelerationStructureToMemoryKHR`, so this is not really a solution.
